src/api/preferencesApi.ts

```diff
@@ -0,0 +1,20 @@
+import { PromotedPublications } from '../types/promotedPublications.types';
+import { PersonPreferencesApiPath } from './apiPaths';
+import { authenticatedApiRequest2 } from './apiRequest';
+
+export const fetchPromotedPublicationsById = async (identifier: string) => {
```

Comment:
Vi prøver å følge en naming convention om at `id` skal være en hel URI mens `identifier` skal være bare UUID. Ser ut som det sendes inn en hel URI her, så kan være en ide å endre navnet her også

```suggestion
export const fetchPromotedPublicationsById = async (id: string) => {
```